supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesJobPaths.scala

```diff
@@ -28,6 +28,10 @@ case class TesJobPaths private[tes] (override val workflowPaths: TesWorkflowPath
   val callInputsDockerRoot = callDockerRoot.resolve("inputs")
   val callInputsRoot = callRoot.resolve("inputs")
 
+  // This is the root directory that TES will use for files related to this task.
+  // We must specify it in the backend parameters sent to TES using "internal_path_prefix" as the key.
```

Comment:
We should call out here that this isn't a standard TES feature, and that it's related to the Azure TES implementation we're working with in Terra.

This brings up the question: should we always include this option or should it be configurable? It's not doing any harm to pass it, but it's yet another Azure special sauce thing.